geode-core/src/test/java/org/apache/geode/distributed/internal/membership/MembershipDependenciesJUnitTest.java

```diff
@@ -20,65 +20,29 @@
 
 import com.tngtech.archunit.core.importer.ImportOption;
 import com.tngtech.archunit.junit.AnalyzeClasses;
-import com.tngtech.archunit.junit.ArchIgnore;
 import com.tngtech.archunit.junit.ArchTest;
 import com.tngtech.archunit.junit.ArchUnitRunner;
 import com.tngtech.archunit.junit.CacheMode;
 import com.tngtech.archunit.lang.ArchRule;
 import org.junit.runner.RunWith;
 
-
 @RunWith(ArchUnitRunner.class)
 @AnalyzeClasses(packages = "org.apache.geode.distributed.internal.membership.gms..",
     cacheMode = CacheMode.PER_CLASS,
     importOptions = ImportOption.DoNotIncludeArchives.class)
 public class MembershipDependenciesJUnitTest {
 
   /*
-   * This test verifies that the membership component (which is currently made up of classes
-   * inside the geode-core module, but which may someday reside in a separate module)
-   * depends only on packages within itself (within the membership component) or packages
-   * outside apache.geode.
-   *
-   * For purposes of this test, classes in the membership...adapter package are not considered.
-   * They will eventually become part of geode-core.
+   * This test verifies that packages defined in the geode-membership module depend only on
+   * packages defined within that module or on packages defined outside the geode-core module
+   * (org.apache.geode packages) or on packages defined in a handful of small "leaf" modules.
    *
-   * While this rule is ignored, comment-out the ignore annotation to run it periodically to get
-   * the current count of deviations.
-   */
-  // TODO: remove ignore once membershipDoesntDependOnCoreProvisional matches this rule exactly
-  @ArchIgnore
```

Comment:
Now that we have eliminated all the spurious dependencies (except the test package one), this `ArchRule` is no longer needed as a baseline.